src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public int getRandomNumber() {
+        int num = new Random().nextInt(100);
+        return num;
+    }
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(colorSupplier.getRandomColor());
+        ball.setNumber(getRandomNumber());
+        return ball;
+    }

```

Comment:
1. Random should be class variable in order not to create new object each time method is called
2. Separate method for 1 command is excessive
3. magic number should be extracted to constant with proper naming
4. create all-arguments constructor for Ball to make code more tidy
```suggestion
    private static final int NUMBER_LIMIT = 100;

    private ColorSupplier colorSupplier = new ColorSupplier();
    private Random random = new Random();

    public Ball getRandomBall() {
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(NUMBER_LIMIT));
    }
```